components/list/list-item-drag-drop-mixin.js

```diff
@@ -339,6 +339,8 @@ export const ListItemDragDropMixin = superclass => class extends superclass {
 					opacity: 0;
 				}
 				:host([selected]) d2l-list-item-drag-handle,
+				:host([current]) d2l-list-item-drag-handle,
+				:host([_focusing-primary-action]) d2l-list-item-drag-handle,
```

Comment:
We want the drag item to be always visible when item is `current`.
We also want it always visible when focus is on the primary action. Enquiring as to if we need to flag this and if releasing it in 20.25.7 is ok. If not, we can flag it and copy this code into `list-item-nav-button-mixin` so that Lessons gets it from the start.